[2021-07-08 13:23:39,463] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: bluepi_dag.query_users_data 2021-07-08T13:23:37.351131+00:00 [queued]>
[2021-07-08 13:23:39,487] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: bluepi_dag.query_users_data 2021-07-08T13:23:37.351131+00:00 [queued]>
[2021-07-08 13:23:39,487] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2021-07-08 13:23:39,487] {taskinstance.py:1088} INFO - Starting attempt 1 of 1
[2021-07-08 13:23:39,487] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2021-07-08 13:23:39,499] {taskinstance.py:1107} INFO - Executing <Task(PostgresOperator): query_users_data> on 2021-07-08T13:23:37.351131+00:00
[2021-07-08 13:23:39,508] {standard_task_runner.py:52} INFO - Started process 18509 to run task
[2021-07-08 13:23:39,515] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'bluepi_dag', 'query_users_data', '2021-07-08T13:23:37.351131+00:00', '--job-id', '125', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/bluepi_dag.py', '--cfg-path', '/tmp/tmpcozxqm5w', '--error-file', '/tmp/tmpyj1at6fr']
[2021-07-08 13:23:39,523] {standard_task_runner.py:77} INFO - Job 125: Subtask query_users_data
[2021-07-08 13:23:39,582] {logging_mixin.py:104} INFO - Running <TaskInstance: bluepi_dag.query_users_data 2021-07-08T13:23:37.351131+00:00 [running]> on host 3cf0296804bd
[2021-07-08 13:23:39,695] {taskinstance.py:1302} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=natchanon
AIRFLOW_CTX_DAG_ID=bluepi_dag
AIRFLOW_CTX_TASK_ID=query_users_data
AIRFLOW_CTX_EXECUTION_DATE=2021-07-08T13:23:37.351131+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-07-08T13:23:37.351131+00:00
[2021-07-08 13:23:39,697] {postgres.py:68} INFO - Executing: 
            SELECT * FROM users;
        
[2021-07-08 13:23:39,708] {base.py:78} INFO - Using connection to: id: my_postgres_conn. Host: 35.247.174.171, Port: 5432, Schema: postgres, Login: exam, Password: ***, extra: {}
[2021-07-08 13:23:40,897] {dbapi.py:204} INFO - Running statement: 
            SELECT * FROM users;
        , parameters: None
[2021-07-08 13:23:41,290] {dbapi.py:212} INFO - Rows affected: 2
[2021-07-08 13:23:41,498] {taskinstance.py:1211} INFO - Marking task as SUCCESS. dag_id=bluepi_dag, task_id=query_users_data, execution_date=20210708T132337, start_date=20210708T132339, end_date=20210708T132341
[2021-07-08 13:23:41,529] {taskinstance.py:1265} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-07-08 13:23:41,575] {local_task_job.py:151} INFO - Task exited with return code 0
